pkg/db/v3/id.go

```diff
@@ -0,0 +1,25 @@
+package v3
+
+import (
+	"time"
+)
+
+type ID struct {
```

Comment:
nit: It'd be helpful to document this type to explain how it's intended to be used. (For example, to answer questions like "Is this an ID that can be used for _any_ data records?")

Reply:
added comments for ID in v1, v2, and v3 as well as a few other structs and fields.